src/serializers/index.js

```diff
@@ -5,11 +5,12 @@ import * as plugins from './plugins';
 import * as protocol from './protocol';
 import OperationSerializer from './operation';
 import opWrapper from './protocol/opWrapper';
+import * as approvalDelta from './wallet';
```

Comment:
Invalid input format. To use approvalDelta i will need to use something like:
```ts
import { serializers } from "echojs-lib"
serializers.approvalDelta.approvalDelta
```
Change name of this import to wallet to use this serializer like:
```ts
import { serializers } from "echojs-lib"
serializers.wallet.approvalDelta
```

Reply:
And add changes of this files to declaration files